sites/fast-website/src/app/data/feature.data.ts

```diff
@@ -1,65 +1,133 @@
+interface FeatureLink {
+    anchorText: string;
+    url: string;
+}
+
 export interface FeatureCardData {
-    body: string;
-    documentationLink: string;
-    githubLink: string;
     header: string;
+    body: string;
+    links?: FeatureLink[];
 }
 
 export const featureCardData: FeatureCardData[] = [
     {
+        header: "Composition",
         body:
             "Create new component compositions by nesting, styling base components, and passing unhandled props. The combinations are endless.",
```

Comment:
This text will need to update, let me work on this.

Reply:
Maybe something along the lines of, "Leverage the power and flexibility of FAST web components to easily and efficiently compose your site your way."